app/src/main/AndroidManifest.xml

```diff
@@ -52,7 +52,8 @@
             android:name=".content.task.StarActivity"
             android:screenOrientation="fullSensor" />
         <activity android:name=".content.task.LessonCompleteActivity" />
-        <activity android:name=".authentication.StudentImageCollectionActivity" />
+        <activity android:name=".authentication.StudentImageCollectionActivity"
+            android:screenOrientation="landscape"/>
```

Comment:
@jogrimst  at the moment yes. If we want to change it we would also need to change the animalTemplate implementation (not a huge task but it's a change)